languages/ig.json

```
@@ -88,6 +89,8 @@
     "We will look into the issue immediately.": "Anyị ga-eleba anya n'okwu a ozugbo.",
     "We'll resolve your issue as quickly as we can.": "Anyị ga-edozi okwu gị ozugbo anyị nwere ike.",
     "Your computer or network has sent too many requests.": "Kọmputa ma ọ bụ netwọkụ gị ezigala ọtụtụ arịrịọ.",
+    "Zoom in": "Bugharịa n&#39;ime",
```

```suggestion
    "Zoom in": "Bugharịa n'ime",
```

We need to automate this replacement, it appears to be the only encoding issue that regularly occurs.